vllm/engine/arg_utils.py

```diff
@@ -176,6 +178,15 @@ def add_cli_args(
                             type=int,
                             default=EngineArgs.max_paddings,
                             help='maximum number of paddings in a batch')
+        parser.add_argument('--scheduler-policy',
+                            type=str,
+                            default=EngineArgs.scheduler_policy,
+                            choices=['fcfs', 'throughput'],
+                            help='scheduler policy')
+        parser.add_argument('--scheduler_max_delay',
+                            type=float,
+                            default=EngineArgs.scheduler_max_delay,
+                            help='scheduler max delay')
```

Comment:
1. change to `--scheduler-max-delay` for consistency
2. could you update the help message indicating that this arg is only valid/used when the policy is `throughput`